tests/test_show_version.py

```diff
@@ -0,0 +1,127 @@
+import pathlib
+
+import pytest
+
+from komodo.show_version import (
+    get_version,
+    guess_komodo_path,
+    guess_komodoenv_path,
+    guess_release,
+    parse_args,
+    read_config,
+)
+
+
+def test_read_config_file(mock_config_file):
+    """Test the function that reads sectionless INI files."""
+    config = read_config("foo.cfg")
+    mock_config_file.assert_called_once_with("foo.cfg", "rt", encoding="utf-8")
+    assert config["home"] == "/foo/bar/komodo-release-0.0.1/root/bin"
+
+
+def test_guess_release(mock_komodo_env_vars):
+    """Simply reads an environment variable."""
+    assert guess_release() == "komodo-release-0.0.1"
+
+
+def test_guess_release_fails_without_env_var():
+    """
+    Check that the script exists gracefully if not run in a komodo environment.
+    """
+    with pytest.raises(SystemExit) as excinfo:
+        _ = guess_release()
+    assert "No active komodo environment found." in str(excinfo.value)
+
+
+def test_guess_komodo_path(mock_komodo_env_vars):
+    """
+    This is the 'standard' scenario when using a vanilla komodo (not komodoenv)
```

Comment:
In this file also there are some uses of '' that seems to be unnecessary.

Reply:
I guess this is me continuing to struggle with emphasizing that there are two different scenarios. Maybe I should move the `komodoenv` tests to another file, that might help.

Reply:
(There are 3 scenarios though, because passing in a manifest file is another option. And I don't think we want 3 test files.)